sys/sys/cdefs.h

```diff
@@ -1093,6 +1093,13 @@
 #define	__kerncap
 #endif
 
+#if (defined(_KERNEL) && __has_feature(capabilities)) || \
+    defined(__CHERI_PURE_CAPABILITY__)
+#define __CHERI_USER_ABI 1
```

Comment:
This is fine. I would maybe have left it undefined and used #ifdef instead of #if as I think that's more typical in FreeBSD.